react/components/icons/TemplateIcon.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+const TemplateIcon: React.FC = () => (
+  <div>
```

Comment:
why does this icon includes a div?

Reply:
I'll a comment next to it explaining why it's necessary.